### PR TITLE
config: add community invitation notification builder

### PR DESCRIPTION
**Pull request by rekt-hard**
:heart: Thank you for your contribution!

### Description

depends on https://github.com/inveniosoftware/invenio-communities/pull/967
closes https://github.com/inveniosoftware/invenio-communities/issues/889

Update notification builder config variable in order to send community member invitation notifications.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [ ] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [ ] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [ ] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Comment by rekt-hard** [comment]
Image of the updated invitation request detail page:
image

File: invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html [hunk 1]
```diff
@@ -29,6 +29,7 @@
   </div>
 
   <h2 class="ui header">{{ request.title }}</h2>
+  <h4 class="ui header">{{ request.description|safe }}</h4>
```
Comment (slint) [inline invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html:32]:
```suggestion
  <p class="ui header">{{ request.description|safe }}</p>
```

minor: can you also change it below, for the other macro?

**Review by slint — COMMENTED** [review]
Minor comment on changing the HTML tag.

File: invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html (hunk 1 above)
Comment (rekt-hard) [reply invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html]: done

**Review by max-moser — APPROVED** [review]
LGTM

**Pull request merged**